From 5c1b547bc7fc5255949182370b749c4f022fe2b2 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Thu, 7 Jul 2016 14:39:01 +0300
Subject: [PATCH 0337/1345] debugfs: a8k: add CPU Ext Ctrl registers

commit  75a828348b0c9627473b17dad34974f4455c6aec from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This register is part of the CA-72 system registers
it holds an important bit called "SMP_EN" that defines if
the CPU accepts DVM messages for TLB invalidates.

Change-Id: I1edac0a99a78a5b26abefaed9ef7ef46a49fb04f
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31000
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/misc/mvebu-a8k-debugfs.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/misc/mvebu-a8k-debugfs.c b/drivers/misc/mvebu-a8k-debugfs.c
index 289d9c4..28f4d22 100644
--- a/drivers/misc/mvebu-a8k-debugfs.c
+++ b/drivers/misc/mvebu-a8k-debugfs.c
@@ -60,6 +60,7 @@ struct sysreg_entry {
 DECLARE_RD_WR_FUNC(s3_1_c15_c0_0);
 DECLARE_RD_WR_FUNC(s3_1_c11_c0_2);
 DECLARE_RD_WR_FUNC(s3_1_c11_c0_3);
+DECLARE_RD_WR_FUNC(s3_1_c15_c2_1);
 
 
 #define SYSREG_ENTRY(code, _desc)		\
@@ -77,6 +78,7 @@ struct sysreg_entry {
 	SYSREG_ENTRY(s3_1_c15_c0_0, "L2 Aux Ctrl"),
 	SYSREG_ENTRY(s3_1_c11_c0_2, "L2 Ctrl"),
 	SYSREG_ENTRY(s3_1_c11_c0_3, "L2 Ext Ctrl"),
+	SYSREG_ENTRY(s3_1_c15_c2_1, "CPU Ext Ctrl"),
 };
 
 
-- 
1.7.9.5

